sdcm/tester.py

```diff
@@ -1076,17 +1078,35 @@ def run_stress_cassandra_thread(self, stress_cmd, duration=None, stress_num=1, k
         return cs_thread
 
     def run_stress_thread_bench(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',  # pylint: disable=too-many-arguments,unused-argument
-                                round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False):  # pylint: disable=too-many-arguments,unused-argument
+                                round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False,
+                                stop_test_on_failure=True):  # pylint: disable=too-many-arguments,unused-argument
 
         timeout = self.get_duration(duration)
+        stop_test_on_failure = False if not self.params.get("stop_test_on_stress_failure") else stop_test_on_failure
+
+        if self.params.get("authenticator") and self.params.get("authenticator") == "PasswordAuthenticator":
```

Comment:
Please redo it in the same way as it is done for c-s:

```
        credentials = self.loader_set.get_db_auth()
        if credentials and 'user=' not in stress_cmd:
```

Reply:
fixed